app/src/main/res/layout/row_item_description.xml

```diff
@@ -1,52 +1,83 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
MaterialCardView should be preferred

Reply:
@macgills We don't have a material theme style defined, if I replace it here it throws the error.

Also, we are not using the material card view anywhere else, so I think this should be done on a separate ticket?

Reply:
You are correct, agh AppCompat themes!

I think I have a ticket open to use a theme based dark mode which would mean using a MaterialComponents.DayNight theme